packages/frontend/src/vue-apollo.js

```diff
@@ -2,117 +2,116 @@
 import Vue from 'vue'
 import VueApollo from 'vue-apollo'
 import { createApolloClient, restartWebsockets } from 'vue-cli-plugin-apollo/graphql-client'
-import { SubscriptionClient } from 'subscriptions-transport-ws';
+import { SubscriptionClient } from 'subscriptions-transport-ws'
+import { LocalStorageKeys } from '@/helpers/mainConstants'
```

Comment:
Moved everything to functions here (instead of triggering object creation as a side-effect when importing htis module) cause originally I planned to use different ApolloProvider instances. I ended up not needing to do it, but getting rid of side-effects is good anyway, so kept these changes (everything works the same as before, it's just refactored)